process_request/tests.py

```diff
@@ -97,6 +98,26 @@ def test_deliver_email(self):
             self.assertNotIn("location", mail.outbox[0].body)
             self.assertNotIn("barcode", mail.outbox[0].body)
 
+    @aspace_vcr.use_cassette("aspace_request.json")
+    def test_get_data(self):
+        get_as_data = ProcessRequest().get_data("/repositories/2/archival_objects/1134638")
+        self.assertIsNot(False, get_as_data)
```

Comment:
Instead of asserting it's not False, I'd suggest asserting the return value is a dict
```suggestion
        self.assertTrue(isinstance(get_as_data, dict))
```